packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2386,7 +2386,7 @@ export class LitNodeClientNodeJs extends LitCore {
           params.resourceAbilityRequests.map((r) => r.resource)
         );
     let expiration = params.expiration || LitNodeClientNodeJs.getExpiration();
-    let nonce = this.latestBlockhash || generateNonce();
+    let nonce = this.latestBlockhash!;
```

Comment:
can we add a check here that `nonce` is defined?  This situation shouldnt happen as we set it in the `handshake` but just in case we shouldnt sign the message unless it is defined in this scope.